tests/test_api_enhancements.py

```
@@ -0,0 +1,262 @@
+"""
+Test suite for API enhancements: caching and rate limiting.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)